### DISCUSSION TITLE
Numerical argument is out of domain

### DISCUSSION THREAD

**Discussion by SGURLY** [solectrus/solectrus#2587] [General]
Zuerst einmal: grandioses Dashboard - im vergleich zu meinen bisherigen Grafana boards um Welten besser!

inb4: Ich bin absolut kein Ruby Entwickler und verstehe vermutlich einfach nicht was genau hier passier - leider ist google da auch nicht sehr hilfreich zu mir. Vlt kann es mir hier ja jemand erklären?

Zum grund setup: Ich nutze ein Sungrow system - beziehe meine Daten über ein Python script und spiele sin in Influx ein. 

Das ganze lief jetzt ein paar Tage super! Seit gester Abend renne ich jedoch in folgenden Fehler:

[192.168.178.55]   
I, [2023-12-01T09:43:38.548897 #13]  INFO -- : [192.168.178.55] method=GET path=/404 format=html controller=ErrorsController action=show status=404 allocations=6222 duration=59.10 view=57.47 db=0.00 user_agent=Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/119.0.0.0 Safari/537.36

I, [2023-12-01T09:43:49.919572 #13]  INFO -- : [192.168.178.55] method=GET path=/ format=html controller=HomeController action=index status=302 allocations=325 duration=3.52 view=0.00 db=0.00 location=http://192.168.178.60:3000/inverter_power/now user_agent=Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/119.0.0.0 Safari/537.36

I, [2023-12-01T09:43:50.050283 #13]  INFO -- : [192.168.178.55] method=GET path=/inverter_power/now format=html controller=HomeController action=index status=200 allocations=14653 duration=114.69 view=108.04 db=0.00 user_agent=Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/119.0.0.0 Safari/537.36

I, [2023-12-01T09:43:50.353049 #13]  INFO -- : [192.168.178.55] method=GET path=/charts/inverter_power/now format=html controller=ChartsController action=index status=200 allocations=35200 duration=190.31 view=189.40 db=0.00 user_agent=Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/119.0.0.0 Safari/537.36

I, [2023-12-01T09:43:50.483191 #13]  INFO -- : [192.168.178.55] method=GET path=/stats/inverter_power/now format=html controller=StatsController action=index status=500 allocations=40110 duration=330.82 view=0.00 db=0.00 user_agent=Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/119.0.0.0 Safari/537.36

F, [2023-12-01T09:43:50.486006 #13] FATAL -- : [192.168.178.55]   

[192.168.178.55] ActionView::Template::Error (Numerical argument is out of domain - log):
[192.168.178.55]      5: turbo-stream action="morph" target="stats" children-only= true
[192.168.178.55]      6:   template
[192.168.178.55]      7:     - if timeframe.now?
[192.168.178.55]      8:       = render StatsNow::Component.new calculator:, field:
[192.168.178.55]      9:     - else
[192.168.178.55]     10:       = render StatsRange::Component.new calculator:, timeframe:, field:
[192.168.178.55]  

Solecturs zeigt seitdem nur noch: `500 Internal Server Error`

Es ist dabei egal, auf welche stats ich via URL zugreifen möchte, der Fehler kommt.

**Comment by ledermann** [comment]
Danke für die Blumen - und den Fehlerreport :) 

Ich habe #2588 angelegt. Alles Weitere dort.
